run_linux.go

```diff
@@ -359,26 +359,36 @@ func runSetupBuiltinVolumes(mountLabel, mountPoint, containerDir string, builtin
 			}
 			initializeVolume = true
 		}
-		stat, err := os.Stat(srcPath)
-		if err != nil {
-			if !os.IsNotExist(err) {
-				return nil, err
+		// Check if srcPath is a symlink
+		stat, statErr := os.Lstat(srcPath)
+		// If srcPath is a symlink, follow the link and ensure the destination exists
+		if stat != nil && (stat.Mode()&os.ModeSymlink != 0) {
```

Comment:
If you change the statErr above to err, and then change this line to

if err == nil && (stat.Mode()&os.ModeSymlink != 0) {

Everything will be happy including me. :^)

Reply:
Fixed